ccp4i/tasks/simbad.tcl

```diff
@@ -127,7 +127,7 @@ proc simbad_task_window { arrayname } {
     widget TITLE  -width 65 \
       -command "check_title_line $arrayname TITLE"
  
-  set link_target "http://simbad.readthedocs.io/en/latest/index.html"
+  set link_target "http://simbad.readthedocs.io/en/latest/contents.html"
```

Comment:
This is not particularly useful. This brings users to the documentation for the Python documentation! Hardly useful for non-coders

Reply:
This is outdated! I've already changed it back to index.

Reply:
This should remain `index.html` but that page needs to be better.

Reply:
That's my assessment too